packages/web-features-figuring/src/baseline/core-browser-set.ts

```diff
@@ -0,0 +1,32 @@
+import { Compat } from "../browser-compat-data/compat";
+import { Release } from "../browser-compat-data/release";
+
+export const identifiers = [
+  "chrome",
+  "chrome_android",
+  "edge",
+  "firefox",
+  "firefox_android",
+  "safari",
+  "safari_ios",
+];
+
+export function browsers(compat: Compat) {
+  return identifiers.map((b) => compat.browser(b));
+}
+
+export function lowReleases(compat: Compat) {
+  return browsers(compat).map((b) => b.current());
+}
+
+export function highReleases(compat: Compat) {
+  return browsers(compat)
+    .flatMap((b) => b.releases())
+    .filter(isBaselineHighRelease);
+}
+
+function isBaselineHighRelease(release: Release) {
+  const baselineHighCutoff = new Date();
+  baselineHighCutoff.setMonth(baselineHighCutoff.getMonth() - 30);
```

Comment:
nit: consider defining `30` as a shared const somewhere else, so that we can reuse it where needed, and document it in the code, somewhat like:

```js
// This is the number of months that Baseline considers enough for a feature to go from low to high.
const BASELINE_LOW_TO_HIGH_TIME = 30;
```

Reply:
Fixed with 3cb4e4e7090993a830a36e1031d3b3e319051bb3 👍